pkg/version/version.go

```diff
@@ -3,5 +3,5 @@ package version
 var (
 	// Version contains the version number. This will be replaced by the linker at build
```

Comment:
Sure we can do that ; ideally I'd still advocate for having it set through the linker at some point ... or we add a version.Commit field here that can be set to the commit's SHA through the linker (to make debugging a bit easier)

Reply:
this will get us by https://issues.redhat.com/browse/OCPBUGS-1621
ART will replace the version with build TS so u will get something like 
```
oc get csv -n openshift-sriov-network-operator
sriov-network-operator.4.11.0-202209161807   SR-IOV Network Operator   4.11.0-202209161807
```